geomagio/WebService.py

```diff
@@ -15,21 +17,16 @@ def __init__(self, factory):
 
     def __call__(self, environ, start_response):
         """Implement WSGI interface"""
+        self.query = WebServiceQuery
         # parse params
+        self.query.parse(environ['QUERY_STRING'])
         # fetch data
+        data = self.query.fetch(self.factory)
```

Comment:
- Move the `fetch` method onto the WebService class
- Make `fetch` just return the data
- Add a new `format(query, data)` method on the WebService that for now uses IAGA2002, and eventually can use different formats.